src/file.cpp

```diff
@@ -51,32 +51,32 @@ using namespace rive;
 // Import a single Rive runtime object.
 // Used by the file importer.
 static Core* readRuntimeObject(BinaryReader& reader, const RuntimeHeader& header) {
-    auto coreObjectKey = reader.readVarUint64();
-    auto object = CoreRegistry::makeCoreInstance((int)coreObjectKey);
+    auto coreObjectKey = reader.readVarUintAs<int>();
+    auto object = CoreRegistry::makeCoreInstance(coreObjectKey);
```

Comment:
Separately, I think we should update the core generator to normalize all the values for the integer types we use for type keys, property ids, class ids, etc. They can likely all be unsigned 16-bit integers.

If there are cases where we need a flag "unknown/non-existent" value, we should see what (if any) implications there are in making those "0" (in some spots I see they are returned as -1 but I don't think that's required, in fact I know core defs specifically use 0 for this as all type/property ids start at 1).

Reply:
...so that makeCoreInstance can take a uint16_t instead of an int.